lineman/app/components/group_page/group_previous_proposals_card/group_previous_proposals_card.haml

```diff
@@ -0,0 +1,17 @@
+%section.group-previous-proposals-card{aria-labelledby: 'group-previous-proposals-card__title'}
+  %h2#group-previous-proposals-card__title.lmo-card-heading{translate: 'group_previous_proposals_card.heading'}
```

Comment:
So, @gdpelican what you've proposed would mean renaming proposal_collapsed to previous_proposal, using it everywhere, including on the group_previous_proposals_panel and modifying it according to what's displayed? My hesitation with that is that the group_previous_proposals_panel only has the pie and the title, and the proposals are not links. Modifying it conditionally sounds complex...but maybe it isn't? Would be keen to work with you and @robguthrie to work out the best way...

My first thought was to extract the pie with position (the css for the proposal_collapsed pie and thread_preview is pretty much the same I think). But I don't know how to structure the css so it looks ok by itself (rather than in the context of a proposal preview or thread preview)